docker-compose.yaml

```diff
@@ -1,26 +1,27 @@
 version: '3'
 
 services:
+
   peatio:
     build:
       context: peatio
       dockerfile: Dockerfile
+    restart: always
     depends_on:
       - db
+      - redis
+      - rabbitmq
     ports:
-      - "8000:80"
-    restart: always
+    - "80:8080"
     environment:
       RAILS_ENV: development
 
   db:
-    image: mysql:5.7
-    volumes:
-      - db_data:/var/lib/mysql
+    image: mysql
     restart: always
     environment:
-      MYSQL_ROOT_PASSWORD: changeme
-      MYSQL_DATABASE: peatio-development
```

Comment:
is it possible to use peatio user ?
I would prefer using params I did set
I see db should be peatio_development